students/gyeongjae-ham/westagram/users/views.py

```diff
@@ -0,0 +1,50 @@
+import json
+
+from django.http     import JsonResponse
+from django.views    import View
+
+from .models import User
+from .validation import validator_email, validator_number, validator_password
+
+class SignUpView(View):
+    def post(self, request):
+        data = json.loads(request.body)
+        try:
+            if User.objects.filter(email=data['email']).exists():
+                return JsonResponse({'MESSAGE':'DUPLICATED_EMAIL'}, status=400)
+            if User.objects.filter(name=data['name']).exists():
+                return JsonResponse({'MESSAGE':'DUPLICATED_NAME'}, status=400)
```

Comment:
```suggestion
            if User.objects.filter(email=data['email']).exists():
                return JsonResponse({'MESSAGE':'DUPLICATED_EMAIL'}, status=400)
                
            if User.objects.filter(name=data['name']).exists():
                return JsonResponse({'MESSAGE':'DUPLICATED_NAME'}, status=400)
```
if 문 사이에 공백 축가하여 가독성 높여주세요.